python/ruff/__main__.py

```diff
@@ -3,6 +3,23 @@
 import sysconfig
 from pathlib import Path
 
+RUFF_PATHS = [
+    Path(sysconfig.get_config_var("userbase")) / "bin" / "ruff",
+    Path(sysconfig.get_path("scripts")) / "ruff",
+]
+
+
+def find_ruff_bin() -> Path:
+    """Return the ruff binary path."""
+    for ruff_path in RUFF_PATHS:
+        if ruff_path.is_file():
```

Comment:
This doesn’t work at all on Windows, where the binary has a `.exe` extension (which had been automatically added by `os.spawnv`).